.github/workflows/postgis-check-command.yml

```diff
@@ -0,0 +1,126 @@
+        description: 'The issue-number of the slash command'
+        required: true
+      comment-creation-date:
+        description: 'The slash command creation date'
+        required: true
+      versions:
+        description: 'Wanted postgis versions to check'
+        default: 12-2.5,13-3.3,14-3.3,14-3.5,16-3.5,17-3.5
+        required: false
+
+env:
+  DB_HOST: localhost
+
+jobs:
+  # Convert versions variable from comma delimited string to JSON
+  versions_to_json:
+    name: Convert versions input to JSON
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.convert-to-json.outputs.VERSIONS }}
+    steps:
+      - id: convert-to-json
+        run: |
+          versions=$(echo ${{ inputs.versions }} | jq -R -c 'split(",")')
+          echo "VERSIONS=$versions" >> $GITHUB_OUTPUT
+  
+  # Create a PR comment for the workflow's results
+  create_comment:
+    name: Create comment
+    runs-on: ubuntu-latest
+
+    steps:
+      # Find our bot's comment
+      - name: Find Comment
+        uses: peter-evans/find-comment@v3
+        id: fc
+        with:
+          issue-number: ${{ inputs.issue-number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Postgres and postgis compatibility
+
+      # Create a comment if it doesn't exist
+      - name: Create comment
+        if: steps.fc.outputs.comment-id == ''
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            **Postgres and postgis compatibility:**
+
+      # If a comment exists, update it with the date and time
+      - name: Update comment
+        if: steps.fc.outputs.comment-id != ''
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          body: |
+            **${{ inputs.comment-creation-date }}**
+
+  tests:
+    name: Run Tests
+    needs: [versions_to_json, create_comment]
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [20.x]
+        postgres: ${{ fromJSON(needs.versions_to_json.outputs.versions) }}
+
+    services:
+      postgres:
+        image: postgis/postgis:${{matrix.postgres}}
+        env:
+          POSTGRES_PASSWORD: 1234
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install TS Project dependencies
+        run: npm ci
+
+      - name: Run tests
+        id: test
+        run: npm run test
```

Comment:
we should not run unit tests when checking the DB

Reply:
Fixed.